### ISSUE TITLE
ingested zarr throws an error in the CLI.

### ISSUE THREAD

**Issue by satra** [dandi/dandi-cli#1022]
on the archive an ingested zarr returns a 400.

```
In [59]: req.status_code
Out[59]: 400

In [60]: req.text
Out[60]: '"Zarr archive already ingested"'
```

when the cli tries to upload this, it shows an ERROR in the pyout console.

i think having a local checksum computation option pre-upload would still be a nice option for upload. for new things that the user knows is not on the archive, they can skip this, but for files that they know are there,

**Comment by yarikoptic** [comment]
could there be a complete example?

**Comment by jwodder** [comment]
I believe this was fixed by #1047.
